client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -94,10 +97,30 @@ export class ProductPurchaseFeaturesList extends Component {
 		const {
 			isPlaceholder,
 			plan,
+			currentPlan,
 			planHasDomainCredit,
 			selectedSite,
 			showCustomizerFeature,
+			scheduleId,
 		} = this.props;
+
+		let hasBusinessOnboardingExpired;
+		if ( currentPlan ) {
+			const expiryDateMoment = this.props.moment( currentPlan.expiryDate );
+			const is2YearPlan = selectedSite.plan.product_id === 1028;
```

Comment:
It's probably cleaner to use a constant rather than hard coding the product id. Can something like `PLAN_BUSINESS_2_YEARS` be used? I'd think there's even a function already to check it too, but not sure off hand.